lib/include/net/cne_ether.h

```diff
@@ -0,0 +1,507 @@
+/* SPDX-License-Identifier: BSD-3-Clause
```

Comment:
This looks like new code, not related to moving header files from one directory to another. Can you put in a different commit? It's hard to complete a review with so much code shuffling aaround.

Reply:
This is not new code, but removed from other files and combined into this one file.